app/views/reports/_date_range_fields.html.haml

```diff
@@ -1,3 +1,6 @@
+%li
+  .field-clarifier= t("preposition.between")
 %li= form.input :date_start, label: t("reports.fields.#{controller_name}.date_start", default: t('reports.fields.date_start')), input_html: { value: human_datetime(@date_start, date_only: true), class: 'datepicker' }
-
+%li
+  .field-clarifier= t("support.array.two_words_connector")
```

Comment:
I felt a little weird using it, since there's no `Array` involved. Still I was hoping something similar existed for "between" but I didn't find it.